custom/icds_reports/utils/fix_past_data_due_to_change_in_height_eligibilty.sql

```diff
@@ -0,0 +1,358 @@
+ --                                       ->  Sort  (cost=147618.83..147712.65 rows=37528 width=167)
+ --                                             Sort Key: chm.awc_id, chm.sex, chm.age_tranche, chm.caste, (COALESCE(chm.disabled, 'no'::text)), (COALESCE(chm.minority, 'no'::text)), (COALESCE(chm.resident, 'no'::text))
+ --                                             ->  Parallel Append  (cost=0.56..142963.52 rows=37528 width=167)
+ --                                                   ->  Parallel Index Scan using chm_month_supervisor_id_default_102648 on child_health_monthly_default_102648 chm  (cost=0.56..142775.88 rows=37528 width=167)
+ --                                                         Index Cond: (month = '2018-05-01'::date)
+
+
+
+UPDATE "agg_child_health_2018-05-01_5" agg_child_health
+  SET
+    wasting_moderate = ut.wasting_moderate,
+    wasting_severe = ut.wasting_severe,
+    wasting_normal = ut.wasting_normal,
+    stunting_moderate = ut.stunting_moderate,
+    stunting_severe = ut.stunting_severe,
+    stunting_normal = ut.stunting_normal,
+    height_measured_in_month = ut.height_measured_in_month,
+    weighed_and_height_measured_in_month = ut.weighed_and_height_measured_in_month
+  FROM (
+    SELECT * FROM temp_agg_child_my
+  ) ut
+  WHERE (
+    agg_child_health.awc_id=ut.awc_id and 
+    agg_child_health.month=ut.month and
+    agg_child_health.gender=ut.sex and
+    agg_child_health.age_tranche=ut.age_tranche and
+    agg_child_health.caste=ut.caste and
+    agg_child_health.disabled=ut.coalesce_disabled and
+    agg_child_health.minority = ut.coalesce_minority and
+    agg_child_health.resident = ut.coalesce_resident
+  )
+
+Roll Ups:
+
+UPDATE 
+  "agg_child_health_2018-05-01_4" agg_child_health 
+SET 
+  wasting_moderate = ut.wasting_moderate, 
+  wasting_severe = ut.wasting_severe, 
+  wasting_normal = ut.wasting_normal, 
+  stunting_moderate = ut.stunting_moderate, 
+  stunting_severe = ut.stunting_severe, 
+  stunting_normal = ut.stunting_normal, 
+  height_measured_in_month = ut.height_measured_in_month, 
+  weighed_and_height_measured_in_month = ut.weighed_and_height_measured_in_month 
+FROM 
+  (
+    SELECT 
+      supervisor_id, 
+      gender, 
+      age_tranche, 
+      SUM(wasting_moderate) as wasting_moderate, 
+      SUM(wasting_severe) as wasting_severe, 
+      SUM(wasting_normal) as wasting_normal, 
+      SUM(stunting_moderate) as stunting_moderate, 
+      SUM(stunting_severe) as stunting_severe, 
+      SUM(stunting_normal) as stunting_normal, 
+      SUM(height_measured_in_month) as height_measured_in_month, 
+      SUM(
+        weighed_and_height_measured_in_month
+      ) as weighed_and_height_measured_in_month 
+    FROM 
+      "agg_child_health_2019-01-01_5" agg_child 
```

Comment:
Why is this month different than the other one?

Reply:
Updated